src/darwin/frida-helper-backend-glue.m

```diff
@@ -217,9 +227,12 @@ static gboolean frida_agent_context_init_functions (FridaAgentContext * self, Gu
 static kern_return_t frida_set_debug_state (mach_port_t thread, gconstpointer state, GumCpuType cpu_type);
 static void frida_set_hardware_breakpoint (gpointer state, GumAddress break_at, GumCpuType cpu_type);
 
+static void frida_breakpoint_receive_request (FridaSpawnInstance * self);
+static void frida_breakpoint_send_response (FridaSpawnInstance * self);
```

Comment:
Since these guys are behaving like member functions of `Frida.SpawnInstance`, I'd say they should be moved to right before `on_server_recv()`, and renamed to something like:

```C
static void frida_spawn_instance_receive_breakpoint_request (FridaSpawnInstance * self);
static void frida_spawn_instance_send_breakpoint_response (FridaSpawnInstance * self);
```